include/boost/geometry/strategies/cartesian/cart_intersect.hpp

```diff
@@ -95,6 +97,39 @@ struct relate_cartesian_segments
         return strategy_type();
     }
 
+    template <typename Geometry>
+    struct area_strategy
+    {
+        typedef area::surveyor
+            <
+                typename point_type<Geometry>::type,
+                CalculationType
+            > type;
+    };
+
+    template <typename Geometry>
+    static inline typename area_strategy<Geometry>::type get_area_strategy()
+    {
+        typedef typename area_strategy<Geometry>::type strategy_type;
+        return strategy_type();
+    }
+
+    template <typename Geometry>
+    struct distance_strategy
+    {
+        typedef distance::pythagoras
+            <
+                CalculationType
+            > type;
+    };
+
+    template <typename Geometry>
+    static inline typename distance_strategy<Geometry>::type get_distance_strategy()
+    {
+        typedef typename distance_strategy<Geometry>::type strategy_type;
+        return strategy_type();
+    }
```

Comment:
Are these two strategies (area, distance) used in the intersection strategy? Don't see that. I'm doubting if it really belongs here - we should avoid that intersect becomes the umbrella strategy... Or do I miss something?